interop/xds/server/server.go

```diff
@@ -78,8 +93,89 @@ func (s *testServiceImpl) EmptyCall(ctx context.Context, _ *testpb.Empty) (*test
 }
 
 func (s *testServiceImpl) UnaryCall(ctx context.Context, in *testpb.SimpleRequest) (*testpb.SimpleResponse, error) {
+	response := &testpb.SimpleResponse{ServerId: s.serverID, Hostname: s.hostname}
+
+behavior:
+	for _, headerVal := range getRPCBehaviorMetadata(ctx) {
+		switch {
+		//If the value matches "sleep-<int>", the server should wait the specified number of seconds before resuming behavior matching and RPC processing.
+		case strings.HasPrefix(headerVal, sleepPfx):
+			{
+				sleep, err := strconv.Atoi(headerVal[len(sleepPfx):])
+				if err != nil {
+					return nil, status.Errorf(codes.InvalidArgument, "invalid format for rpc-behavior header %v, must be '%v<int>' instead", headerVal, sleepPfx)
```

Comment:
instead of `must be '%v<int>`, prefer using `must be 'sleep-<int>`. That way, the code is more readable.

Same comment applies to other error strings as well.